contracts/common/misc/ERCProxy.sol

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-License-Identitifer:    MIT
+ */
+
+pragma solidity ^0.5.5;
+
+// See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-897.md
+
+interface ERCProxy {
+  function proxyType() external pure returns (uint256 proxyTypeId);
+  function implementation() external view returns (address codeAddr);
```

Comment:
No `updateProxyTo`? In case of non-upgradable proxy, that will simple throw.

Reply:
We added `updateImplementation`